components/uploader.py

```diff
@@ -13,53 +17,56 @@ def get_component():
             children=html.Div([
                 'Drag and Drop or ',
                 html.A('Select Files')
-            ]),
-            style={
-                'width': '95%',
-                'height': '60px',
-                'lineHeight': '60px',
-                'borderWidth': '1px',
-                'borderStyle': 'dashed',
-                'borderRadius': '5px',
-                'textAlign': 'center',
-                'margin': '2.5%'
-            }
+            ])
         ),
-        dcc.Loading(
-            children=html.Div(id='output-data-upload')
-        )
+        html.Div([
+            html.Div(
+                dbc.Button(
+                    'Use sample catalog', color='primary'
+                ),
+                id='sample-dataset',
+                title='Use the sample SCEDC 2018 Focal Mechanism catalog'
+            ),
+            dcc.Loading(
+                id='output-data-upload',
+                className='alert-message'
+            )
+        ], id='output-container')
     ])
 
 
-def update_output(contents, filename, session_id):
+def update_output(contents, filename, session_id, use_sample_data=False):
     """Return a success or an error message depending on the success
     of parsing.
 
     Keyword arguments:
     contents -- The contents of the uploaded file as a binary string
     filename -- Name of the uploaded file
     session_id -- ID of the current session
+    use_sample_data -- Whether the sample data set should be used
     """
+    if use_sample_data:
+        filename = 'sc2018_hash_ABCD_so.focmec.scedc'
```

Comment:
Maybe put the default file name as a global variable on top of this file so that it can be easily found and changed if necessary?